### ISSUE TITLE
Pixi: for transformed doucments check that all amp-element layouts are ssr'd 

### ISSUE THREAD

**Issue by sebastianbenz** [ampproject/amp.dev#5468]
We should show an error for the following doc:

```
<html amp i-amphtml-layout i-amphtml-no-boilerplate transformed="self;v=1">

<amp-ad data-block-on-consent="_till_responded" layout="fixed" width="320" height="100"
  type="doubleclick" data-slot="..."
  data-multi-size="320x50" data-multi-size-validation="false">
  </amp-ad>
```

as the amp-ad is missing:

```
class="i-amphtml-layout-fixed i-amphtml-layout-size-defined"
```

I think checking if each AMP element with a layout contains `layout="$LAYOUT" class="i-amphtml-layout-$LAYOUT`.

**Comment by kristoferbaxter** [comment]
Could we also add logic for the cases where inline styles should be present?

So for instance this element should also have `style="width:320px;height:100px"`.

**Comment by sebastianbenz** [comment]
We could, I'm not sure how useful it's going to be. My assumption is that optimisers are usually correct (the open source ones use the same test suite). So the most important part is to check whether the element has been transformed or not.

**Comment by kristoferbaxter** [comment]
Makes sense, checking for all three here would prevent the CLS issues found in the wild... but perhaps this isn't as widespread of an issue.
